resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/submit/KubernetesClientUtilsSuite.scala

```diff
@@ -82,8 +81,9 @@ class KubernetesClientUtilsSuite extends SparkFunSuite with BeforeAndAfter {
   }
 
   test("verify that configmap built as expected") {
-    val configMapName = "testConfigMapName"
-    val properties = Map(Config.KUBERNETES_NAMESPACE.key -> "default")
+    val configMapName = java.util.UUID.randomUUID.toString
```

Comment:
`import java.util.UUID`?

Reply:
It works without "import java.util.UUID". Is this a code import style limitation? If true, I can import :)

Reply:
@dcoliversun . Apache Spark recommends not to hide your import in the code. Especially, you are using twice at line 84 and 85. Your code is verbose due to the repeatability and not good at the readability too.

Reply:
@dongjoon-hyun Thanks for your suggestion. I import java.util.UUID and add name/namespace prefix to improve readability. Please reivew it 😊